src/cast_core/test/CommitLatency_test.cpp

```diff
@@ -0,0 +1,73 @@
+// Copyright 2019-present MongoDB Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <bsoncxx/json.hpp>
+#include <bsoncxx/builder/stream/document.hpp>
+
+#include <boost/exception/diagnostic_information.hpp>
+
+#include <yaml-cpp/yaml.h>
+
+#include <testlib/MongoTestFixture.hpp>
+#include <testlib/ActorHelper.hpp>
+#include <testlib/helpers.hpp>
+
+#include <gennylib/context.hpp>
+
+namespace {
+using namespace genny::testing;
+namespace bson_stream = bsoncxx::builder::stream;
+
+TEST_CASE_METHOD(MongoTestFixture, "CommitLatency successfully connects to a MongoDB instance.",
+          "[standalone][single_node_replset][three_node_replset][sharded][CommitLatency]") {
+
+    dropAllDatabases();
+    auto db = client.database("mydb");
+
+    YAML::Node config = YAML::Load(R"(
+        SchemaVersion: 2018-07-01
+        Actors:
+        - Name: CommitLatency
+          Type: CommitLatency
+          Database: mydb
+          ExecutionStrategy:
+            ThrowOnFailure: true
+          Phases:
+          - Repeat: 100
+            Collection: mycoll
+            Document: {foo: {^RandomInt: {min: 0, max: 100}}}
+    )");
+
+
+    SECTION("Inserts documents into the database.") {
```

Comment:
I think you can just delete this test file unless you plan to fill it in later (which of course would be a <i>good idea</i>™️ but it's your call)

Reply:
If management allows me to spend the time, I'd rather like to write some unit tests. Part of the motivation for this was also to learn genny.